dist/jquery.stickybits.js

```diff
@@ -65,59 +65,58 @@ Stickybit.prototype.setStickyPosition = function setStickyPosition() {
   => based on window scroll
 */
 Stickybit.prototype.manageStickiness = function manageStickiness() {
-  var parent = this.el.parentNode;
-  var pv = this.positionStickyVal;
+  // cache variables
+  var el = this.el;
+  var parent = el.parentNode;
+  var pv = this.positionVal;
   var vp = this.vp;
```

Comment:
The variable noted above _was _used. Current linting would catch unused variables. 🤓